### DISCUSSION TITLE
Support for plain Fetch API

### DISCUSSION THREAD

**Discussion by Timkor** [nuxt-community/auth-module#1481] [Ideas & Feature Requests]
### Is your feature request related to a problem? Please describe.

Users might not use `axios` or `@nuxt/http` - particularly with the new Nuxt polyfill for fetch.

It would be good to support the plain Fetch API, since it's already polyfilled by Nuxt.

### Describe the solution you'd like to see
Do not load any `networkHandler` if set to false and default to the Fetch API.
```
auth: {
  networkHandler: false,
},
```

**Comment by atinux** [comment]
I think it's planned for the v5 already.

**Comment by nathanchase** [comment]
Or what about the ability to use https://github.com/nuxt-contrib/ohmyfetch instead of axios?

**Reply by TomMiller-mas** [reply]
I would second this request.  This is "native" to the project, so it should be an easy decision to support ohmyfetch.

**Reply by joffreyBerrier** [reply]
@nathanchase Any news about that ?

**Reply by nathanchase** [reply]
> @nathanchase Any news about that ?

Only current option is https://github.com/Teranode/nuxt-module-alternatives

**Comment by innovaweb-dev** [comment]
I use GraphQL, what about this option ?

**Comment by bmulholland** [comment]
GraphQL (at least via Apollo) is covered in https://github.com/nuxt-community/auth-module/issues/72. Non-apollo will need a separate item to cover it. On the nuxt side, there's some plans to change how network requests happen to remove the axios requirement, and this will happen after nuxt 3 is out.
